outputManager.go

```
@@ -0,0 +1,110 @@
+			account.Email,
+			account.Name,
+			account.Status,
+			account.JoinedMethod,
+			account.JoinedTimestamp,
+		})
+		if err != nil {
+			return fmt.Errorf("error appending table row: %w", err)
+		}
+	}
+
+	table.Render()
```

golangci-lint (1.64.8)

Error return value of `table.Render` is not checked

(errcheck)